### ISSUE TITLE
pykwalify schema considers typetokens invalid

### ISSUE THREAD

**Issue by tommilligan** [taverntesting/tavern#153]
When trying to use the pykwalify validated schema with type converted variables, a `BadSchemaError` is thrown. For example:
```yaml
variables:
  verify-ssl: False

---

test_name: Test health
stages:
- name: health
  request:
    url: ...
    method: POST
    json:
      ...
    verify:
      value: !bool "{verify-ssl}"
```
will throw something similar to
```
E   pykwalify.errors.SchemaError: <SchemaError: error code 2: Schema validation failed:        
E    - Value '<tavern.util.loader.BoolToken object at 0x7f388526c4a8>' is not of type 'bool'. Path: '/stages/0/request/verify'.: Path: '/'>
```

**Comment by tommilligan** [comment]
Added failing tests in PR #154, will look at possible fixes this week

**Comment by michaelboulton** [comment]
Fixed in #154
